docs/workflow/blocks/load-schema.rst

```diff
@@ -0,0 +1,117 @@
+Load schema  
+===========
+
+The Load Schema block is used to load a schema from the "metadata" database and 
+transform it to a jsonSchema which can be used to define a dynamic form. 
+
+Typical usage
+-------------
+Used as a sub-block of the "Form" schemaGetter definition. 
+
+.. code-block:: javascript
+
+{
+    "type": "form",
+    "label": "Save",
+    "hasSubmit": false,
+    "schemaGetter": {
+        "blocks": [
+            {
+                "type": "load-schema",
+                "adapterName": "schemas",
+                "schemaGetter": "context.schema && context.schema.name"
+            }
+        ]
+    }
+}
+
+
+Default config
+--------------
+
+.. code-block:: javascript
+
+    {
+      "type": "load-schema",
+      adapterName: 'schemas',
+      '//comment': 'use either schema or schemaGetter',
+      schema: 'schema',
+      schemaGetter: 'data.schema',
+    }
```

Comment:
This seems conflicting.
The "default config" example is confusing because there is no block in the "Select Task" GUI that actually produces a block with that config.
How about 2 seperate examples without mentioning what is default?